DESCRIPTION

```diff
@@ -42,6 +42,7 @@ Imports:
     readxl,
     rlang (>= 0.3.1),
     sf (>= 0.7),
+    tidyselect,
```

Comment:
Should we have a version here? `dplyr` switched to tidyselect at [version 0.7.5](https://github.com/tidyverse/dplyr/releases/tag/v0.7.5). Wondering if we should set that, and have `tidyselect >= 0.2.5)`
```suggestion
    tidyselect (>= 0.2.5),
```